### ISSUE TITLE
httpx.ReadError: [SSL: TLSV13_ALERT_CERTIFICATE_REQUIRED] tlsv13 alert certificate required for version 0.28.0

### ISSUE THREAD

**Issue by Rai220** [encode/httpx#3441]
I am using 3 parameters to make request:
cert=(cert.crt, crt.key)
verify=False

In version 0.27.2 everything is OK.

**Comment by tomchristie** [comment]
Interesting thanks. Does the above PR look correct to you?

**Comment by Rai220** [comment]
No, nothing changes.

**Comment by hbldh** [comment]
I experience this as well after upgrading to 0.28.0. But I had a certificate sent in as a string path for the `verify` keyword, as well as a client cert:

```python
import httpx

certs = ...        # Two pem files read from disk
server_cert = ...  # Another pem file read from disk
client = httpx.Client(cert=certs, verify=str(server_cert))
client.post("/myendpoint", json={})
```

This gives the error reported above.

When I install the certificate in my Certificate Store (I am using Windows) and remove the `verify=str(server_cert)` I get this error instead:

```
httpx.ConnectError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)
```

Has something changed in how certificates are loaded?

**Comment by hbldh** [comment]
My issue has been resolved with 0.28.1. Thank you very much for the rapid response on this!
